WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -614,24 +615,16 @@ private extension ProductFormViewController {
     }
 
     func configureTooltipPresenter() {
```

Comment:
nit: we usually use `configure*` functions for one-time setup, while this can be called resetting `tooltipPresenter` more than once. maybe `update*` can be more clear?
```suggestion
    func updateTooltipPresenter() {
```

Reply:
Done in e858ff7a7b1eb24c11464dc3e75cbd08e09277fb